src/app/charts/core/files/secrets/tpl/arkcase-server.yaml

```diff
@@ -139,3 +139,6 @@ notification:
   arkcase.port: {{ $baseUrl.port }}
   arkcase.url: {{ printf "%s://%s" $baseUrl.scheme $baseUrl.hostname | quote }}
   arkcase.url.base: {{ $baseUrl.url | quote }}
+
+ocr:
+  tempPath: "/app/home/temp/tomcat"
```

Comment:
Please replace this value with `${TEMP_DIR}`, for consistency with other bits of code, and to avoid hardcoding paths here.